### ISSUE TITLE
Re-rendering with different HTML y-axis titles causes an unchecked exception

### ISSUE THREAD

**Issue by peteb4ker** [highcharts/highcharts#2472]
**Overview**

I can't commit to generating a fiddle to reproduce this bug as it will take me far too long, but I can point directly to the source of the issue.

**How to reproduce**

I have a chart with 6 series `[a,b,c,d,e,f]`.  I can switch between 5 different views showing `[a,b,c]`, `[d,e,f]` (which use one axis) or `[a,d]`, `[b,e]`, `[c,f]` (which use two axes).  

When using HTML yAxis titles, I see an unchecked exception when switching between the views:

```
Uncaught TypeError: Cannot read property 'childNodes' of undefined highstock.src.js?dev:3016
SVGElement.destroy highstock.src.js?dev:3016
(anonymous function) highstock.src.js?dev:8581
each.Array.forEach.i highstock.src.js?dev:1202
Axis.destroy highstock.src.js?dev:8579
Chart.destroy highstock.src.js?dev:12057
Chart.getContainer highstock.src.js?dev:11399
Chart.firstRender highstock.src.js?dev:12136
Chart.init highstock.src.js?dev:10660
(anonymous function) highstock.src.js?dev:19221
obj.(anonymous function) highstock.src.js?dev:427
(anonymous function) highstock.src.js?dev:19864
obj.(anonymous function) highstock.src.js?dev:427
Chart highstock.src.js?dev:10564
Highcharts.StockChart highstock.src.js?dev:20073
$.fn.highcharts highstock.src.js?dev:1236
```

This is due to the following line (~3016 highstock.js):

```javascript
while (parentToClean && parentToClean.div.childNodes.length === 0) {
```

If `parentToClean` exists but does not have a `div` child, an exception is thrown.  To avoid this problem, the line should be:

```javascript
while (parentToClean && parentToClean.div && parentToClean.div.childNodes.length === 0) {
```

Or it may be possible to shorten this to:

```javascript
while (parentToClean.div && parentToClean.div.childNodes.length === 0) {
```

Either one works to fix this bug, but the shortened version may have other you guys will know best which one is the appropriate fix.

Thanks!

**Comment by sebastianbochan** [comment]
Commit is included in the current version (4.2.6) and problem does not appear. Reopen if will be reproducible.
